### ISSUE TITLE
Remove commission max rate and max change rate during create-validator

### ISSUE THREAD

**Issue by terencesflam** [irisnet/irishub#1242]
**Is your feature request related to a problem? Please describe.**
As both of the commission max rate and max change rate don't indicate the effective commission rate of a validator at a particular moment, the difference in these figures amongst validators may give misleading information to delegators for deciding how to make their delegations.

**Describe the solution you'd like**
I suggest to remove commission max rate and max change rate during create-validator. The max of both these parameters should be align to their default numbers set in the protocol.

**Comment by HaoyangLiu** [comment]
I don't agree with this idea. We provide rainbow wallet app. With this app,  users can get the validator list and all commission related parameters will be shown to users. The effective commission rate will be highlighted to distinguish with other commiission parameters. So it is not necessary to worry about this issue. 

Besides, it is not reasonable to use the same configuration about commission max rate and max change rate on all validators. The effective commission rate can't be changed frequently, the minimum interval is 24 hours. So lowwer commission max rate or max change rate is also a way for validators to attract more delegatioins. 

@terencesflam What do you think about my opinion?

**Comment by terencesflam** [comment]
@HaoyangLiu 

Thanks for the reply. I think this is the effective commission rate which should be varied across different validators as a free market. I agree this can't be changed frequently. But this is only a commercial and operational decision. If the validators change the commission rate frequently, they will lose clients' (ie. delegators') confidence. 

This is reasonable that the max rate and max change rate are having the same maximum in protocol as they don't mean the current effective rate. If the validators would like to use these two parameters to their advantage in the market, this is their commercial decision which they can state in their marketing materials or service agreement (eg. a validator can say "we will never raise our commission rate to above XX%".)

**Comment by terencesflam** [comment]
Let me further elaborate from another perspective. I think allowing validators to set the max rate and max change rate by themselves in the genesis file which cannot be changed forever (which means even after mainnet launch) will cause fatal mistakes which cannot be recovered. For example, now we see that quite some well-known validators have submitted their genesis file with the default parameters, which are max rate 20% and max change rate 1%. We have a strong gut feeling that they have neglected these parameters, or they don't know these parameters cannot be changed even after mainnet (or we are wrong and these can be changed after mainnet?)

From our discussion with other experienced validators, validating on Tezos need a commission rate above 20% just to break even. This tech is new. This market is new. Crypto price is very volatile. What if 20% is still a low rate that the validators cannot earn a profit? These validators will just quit, because they cannot raise the rate to above 20%. IRIS will lose these validators.

The reason why we raise this issue is that we have noticed some other competent validators are also just using the default values (max 20%/ max change 1%) and we feel the pressure to follow while we are doubtful if they have submitted that correctly. Given there are too many uncertainties in this brand new business, why those guys are so confident that they will never need a max commission rate higher than 20%?

In short, I cannot see much value from this setting given there is already a max rate in protocol, which is 100%. But I see a big threat if I am right (that many validators who have submitted the genesis file have neglected to reset these parameters and will face a difficult situation to run a staking business at a loss but cannot change the max rate in the future).

**Comment by haifengxi** [comment]
We'll have the request implemented properly before betanet launch, while trying our best to avoid asking  people to re-submit their gentx's. @terencesflam

**Comment by wukongcheng** [comment]
- Ignore the params in gentx
- Make the`max change rate` param can be gov
